mffpy/xml_files.py

```diff
@@ -1,3 +1,13 @@
+import copy
+from .epoch import Epoch
+from .dict2xml import TEXT, ATTR
+from cached_property import cached_property
+from typing import Tuple, Dict, List, Any, Union, IO
+import numpy as np
+from collections import defaultdict
+from datetime import datetime
+import xml.etree.ElementTree as ET
+import logging
 """
```

Comment:
That's a good addition.  I wanted to make you aware that PEP-257 specifies that:

> A docstring is a string literal that occurs as the first statement in a module, function, class, or method definition. Such a docstring becomes the __doc__ special attribute of that object.
(from https://www.python.org/dev/peps/pep-0257/)

Accordingly, the module docstring should be positioned before the imports.  I would structure like so:
```python
"""
<module name and one liner>

<additional information>
""" 

# Here comes the module imports, code etc.

"""
<Copyright notice at the very end>
"""
```

The idea is that if one opens a file one almost never looks for the copyright notice.  Its presence is of purely legal relevance.  Therefore, it makes sense to append this part of the doc.

Reply:
I put them both at the top of the file to stick with the convention of the other files in this repository. At some point we could format all of the files to put the copyright at the end.